src/Augmentation/ui/PlayerMultipliers.tsx

```diff
@@ -23,14 +23,22 @@ interface IBitNodeModifiedStatsProps {
   color: string;
 }
 
+function customFormatPercent(value: number): string {
+  if (value >= 100) {
```

Comment:
I don't get the purpose of this `if`. It's duplicating a check that happens inside `formatPercent`; shouldn't `formatPercent(value, 2, 100)` work the same unconditionally?